newIDE/app/src/UI/ClosableTabs.js

```diff
@@ -191,24 +194,28 @@ export function ClosableTab({
                 </ButtonBase>
               )}
             </span>
-            <ContextMenu
-              ref={contextMenu}
-              buildMenuTemplate={() => [
-                {
-                  label: 'Close',
-                  click: onClose,
-                  enabled: closable,
-                },
-                {
-                  label: 'Close others',
-                  click: onCloseOthers,
-                },
-                {
-                  label: 'Close all',
-                  click: onCloseAll,
-                },
-              ]}
-            />
+            <I18n>
+              {({ i18n }) => (
+                <ContextMenu
+                  ref={contextMenu}
+                  buildMenuTemplate={(i18n: I18nType) => [
```

Comment:
Here and in other places, I see that you've both been importing `<I18n>` and pass `i18n` as the buildMenuTemplate parameter.

I think we have to choose between one or the other, but not have both.
To make the code simpler, I'm in favor of passing `i18n` as the parameter of `buildMenuTemplate`. This allows to avoid adding `<I18>` everywhere (apart from where you need `i18n` outside of a context menu).
It also go in the direction of one day having `label` taking just a `t` macro, without the need to call `i18n._` (leaving this job to ContextMenu.. but it implies extra work to make this work on Electron).

See my next comment about how to add it to ContextMenu for Electron and material :)